README.md

```diff
@@ -1,23 +1,23 @@
-# TV Kitchen Appliance Core
+[![Build Status](https://travis-ci.com/tvkitchen/appliances.svg?branch=master)](https://travis-ci.com/tvkitchen/base)
```

Comment:
You changed one of them but not the other!

Reply:
Oh.............. oh no.

HOTFIX TIME!